oe_theme.theme

```diff
@@ -529,6 +531,32 @@ function oe_theme_preprocess_field(&$variables, $hook): void {
       ];
     }
   }
+  if ($variables['element']['#field_type'] === 'oe_featured_media') {
```

Comment:
Might be better to provide field formatter? In this case we can re-use it in Contact entity, for example.

Reply:
yes, your right @yenyasinn